packages/paste-website/src/pages/foundations/localization/index.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: UX Localization Guidelines
+description: Localization stands for adapting a product, service or content to fit a specific market or country.
+slug: /foundations/localization/
+---
+
+import {graphql} from 'gatsby';
+import Img from 'gatsby-image';
+
+import {Button} from '@twilio-paste/button';
+import {Anchor} from '@twilio-paste/anchor';
+import {Stack} from '@twilio-paste/stack';
+import {DeleteIcon} from '@twilio-paste/icons/esm/DeleteIcon';
+
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+
+import {SidebarCategoryRoutes} from '../../../constants';
+
+export const pageQuery = graphql`
+  {
+    mdx(fields: {slug: {eq: "/foundations/localization/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<content>
+
+<ComponentHeader
+  name={props.pageContext.frontmatter.title}
+  description={props.pageContext.frontmatter.description}
+  categoryRoute={SidebarCategoryRoutes.FOUNDATIONS}
+/>
+
+</content>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+## Introduction
+
+Self-service tips on how to build experiences in a way that’s appropriate and meaningful to all our customers, no matter where they are in the world.
+
+### Definition of terms
+
+<h4 aria-describedby="internationalization">Internationalization</h4>
+<p id="internationalization">
```

Comment:
Question: why are you adding the HTML tags and id's manually? Usually we just let markdown handle it all for us.

Reply:
Oh I didn't remove the `aria-describedby` attributes 🤦‍♀️ will do now.

I only added an `id` to the paragraph elements because I wanted to add the attribute to the headers.

Reply:
Ok, cool! Yeah its not necessary to do that.